### DISCUSSION TITLE
Is it possible to have more than 2 characters in FileDecorationProvider badge?

### DISCUSSION THREAD

**Discussion by alekmeckaroski** [microsoft/vscode-discussions#1027] [Extension Development QnA] [answered]
Hi all,

My team is working on vscode extension and we have a  need to implement a badge with some count in a treeview items.
Now, we are using FileDecorationProvider in the TreeItem, but the badge inside can't hold more than 2 characters.
We want to display count number that can be higher than 99.

How can we do this?

**Comment by isidorn** [comment, marked as answer]
Not supported right now in the UI. I suggest to open a new feature request in the vscode repo. Thank you
